src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1847,6 +1955,10 @@ internal static string GetAssemblyFileVersion(Assembly assembly)
 
         protected override void OnHostInitialized()
         {
+            string message = $"Host initialized ({_stopwatch.ElapsedMilliseconds}ms)";
```

Comment:
This first log point is the earliest at which the host will accept http traffic. Listeners may not yet be started.